tests/oss-fuzz/README.md

```diff
@@ -2,19 +2,48 @@
 
 This document provides links and knowledge about fuzzing libavif on oss-fuzz.
 
+# Fuzzing AVIF locally
+
+You need to build with the following CMake flags:
+
+```sh
+-DAVIF_CODEC_AOM=LOCAL -DAVIF_CODEC_AOM_DECODE=ON -DAVIF_CODEC_AOM_ENCODE=ON -DAVIF_CODEC_DAV1D=LOCAL -DAVIF_LIBYUV=LOCAL -DAVIF_LIBSHARPYUV=LOCAL -DAVIF_BUILD_TESTS=ON -DAVIF_ENABLE_GTEST=ON -DAVIF_GTEST=LOCAL -DAVIF_LOCAL_FUZZTEST=ON -DAVIF_ENABLE_FUZZTEST=ON -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DAVIF_ENABLE_WERROR=OFF
+```
+
+You can then run your tests as normal:
+
+```sh
+ctest --output-on-failure -R avif_fuzztest_enc_dec
```

Comment:
maybe just use 'fuzztest' instead of 'avif_fuzztest_enc_dec' to run all fuzzers in this example command line